CondCore/ESSources/python/PoolDBESSource_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-from CondCore.DBCommon.CondDBSetup_cfi import *
+from CondCore.CondDB.CondDB_cfi import *
 
```

Comment:
CondCore.CondDB.CondDB_cfi does not contain a "CondDBSetup" PSet needed later in this cfi file.
It does include a CondDB PSet. Should the latter be use, or should the above change be reverted?
(75X/76X).